package.json

```diff
@@ -531,6 +571,22 @@
           "when": "resourceScheme =~ /webview/ && r.helpPanel.active",
           "group": "navigation"
         }
+      ],
+      "view/item/context": [
+        {
+          "command": "r.workspaceViewer.view",
+          "group": "inline"
+        }
+      ],
```

Comment:
Is it useful to add a `r.workspaceViewer.remove` which sends `rm(<symbol>)` so that user could remove a symbol from the global environment?

Reply:
Added remove in c3f3c17

![image](https://user-images.githubusercontent.com/60372411/100562615-1c959800-32b4-11eb-9e9f-04070dc73a0a.png)